examples/events.py

```diff
@@ -30,22 +30,25 @@
     container = await docker.containers.create_or_replace(
         config=config, name='testing')
     await container.start()
-    print(f"=> created and started container {container._id[:12]}")
+    print("=> created and started container {}".format(container._id[:12]))
 
     while True:
         event = await subscriber.get()
         if event is None:
             break
-        print(f"event: {event!r}")
+
+        for key, value in event.items():
+            print(key,':', value)
+        #print("event: {}".format(event!r))
```

Comment:
Please clean up the debug comments and update the coding style to add a space after commas (e.g., `key,':',` => `key, ':'`)